applications/tari_base_node/windows/README.md

```diff
@@ -67,15 +42,57 @@ Notes:
           install_sqlite.bat
           install_tor_services.bat
           install_vs2019_redist.bat
+          install_xmrig.bat
```

Comment:
I run into this error when trying to run this script:

```
get-xmrig-win.ps1 cannot be loaded because running  scripts is disabled on this system.
```

Looks like we need to investigate the execution policy further.

Reply:
Yip, still looking into this...

Reply:
This issue has been resolved with 
```
powershell "Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process; .\get-xmrig-win.ps1"
```